sample/src/App.tsx

```diff
@@ -27,8 +27,8 @@ Sentry.init({
   // Replace the example DSN below with your own DSN:
   dsn: SENTRY_INTERNAL_DSN,
   debug: true,
-  beforeSend: (e: Sentry.Event) => {
-    console.log('Event beforeSend:', e);
+  beforeSend: (e, hint) => {
+    console.log('Event beforeSend:', e, 'hint:', hint);
```

Comment:
Are you sure you wanna add the hint back? it caused problems already.

Reply:
I remember removing it. :( Now it's gone.